documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -12,356 +14,156 @@ ifdef::community[]
+The first time it connects to a PostgreSQL server or cluster, the connector takes a consistent snapshot of all schemas. After that snapshot is complete, the connector continuously captures row-level changes that insert, update, and delete database content and that were committed to a PostgreSQL database. The connector generates data change event records and streams them to Kafka topics. For each table, the default behavior is that the connector streams all generated events to a separate Kafka topic for that table. Applications and services consume data change event records from that topic. 
 
+ifdef::product[]
+Information and procedures for using a {prodname} PostgreSQL connector is organized as follows: 
+
+* xref:overview-of-debezium-postgresql-connector[]
+* xref:how-debezium-postgresql-connectors-work[]
+* xref:descriptions-of-debezium-postgresql-connector-data-change-events[]
+* xref:how-debezium-postgresql-connectors-map-data-types[]
+* xref:setting-up-postgresql-to-run-a-debezium-connector[]
+* xref:deploying-and-managing-debezium-postgresql-connectors[]
+* xref:how-debezium-postgresql-connectors-handle-faults-and-problems[]
+
+endif::product[]
 
+// Type: concept
+// Title: Overview of {prodname} PostgreSQL connector 
+// ModuleID: overview-of-debezium-postgresql-connector
 [[postgresql-overview]]
 == Overview
 
-PostgreSQL's https://www.postgresql.org/docs/current/static/logicaldecoding-explanation.html[_logical decoding_] feature was first introduced in version 9.4 and is a mechanism which allows the extraction of the changes which were committed to the transaction log and the processing of these changes in a user-friendly manner via the help of an https://www.postgresql.org/docs/current/static/logicaldecoding-output-plugin.html[_output plug-in_]. This output plug-in must be installed prior to running the PostgreSQL server and enabled together with a replication slot in order for clients to be able to consume the changes.
+PostgreSQL's link:https://www.postgresql.org/docs/current/static/logicaldecoding-explanation.html[_logical decoding_] feature was introduced in version 9.4. It is a mechanism that allows the extraction of the changes that were committed to the transaction log and the processing of these changes in a user-friendly manner with the help of an link:https://www.postgresql.org/docs/current/static/logicaldecoding-output-plugin.html[_output plug-in_]. You must install this output plug-in and configure a replication slot that uses the output plug-in before running the PostgreSQL server. This enables clients to consume the changes. 
```

Comment:
Of course. I might not have understood the implication when I started working on this content, but I certainly knew this by the end. I have update all content so that it is clear that installation of a plug-in might be required only upstream. Downstream, there is no longer content related to installing a plug-in. So this is fixed in the next commit. (subject of course to your review/approval :-)